activitysim/abm/models/disaggregate_accessibility.py

```diff
@@ -164,6 +164,11 @@ class DisaggregateAccessibilitySettings(PydanticReadable, extra="forbid"):
     """
     NEAREST_METHOD: str = "skims"
 
+    postprocess_proto_tables: list[DisaggregateAccessibilityAnnotateSettings] = []
+    """
+    List of preprocessor settings to apply to the proto-population tables after generation.
+    """
+
```

Comment:
This PR actually adds two things to DisaggregateAccessibilitySettings, we also need to add something like this...

```
    KEEP_COLS: list[str] | None = None
    """
    Columns to keep in the proto-population tables after generation.

    If None, all columns that have names containing the word ""accessibility" are kept.
    """
```